2018-03-13 21:52:50.120318: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX
2018-03-13 21:52:50.317347: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties: 
name: GeForce GTX 1070 major: 6 minor: 1 memoryClockRate(GHz): 1.7845
pciBusID: 0000:05:00.0
totalMemory: 7.92GiB freeMemory: 7.60GiB
2018-03-13 21:52:50.317398: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1070, pci bus id: 0000:05:00.0, compute capability: 6.1)
Using TensorFlow backend.
/home/tutysara/anaconda2/envs/dog-project/lib/python3.6/importlib/_bootstrap.py:205: RuntimeWarning: compiletime version 3.5 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.6
  return f(*args, **kwds)
DEBUG:__main__:prediction for fine tune fc layers
DEBUG:__main__:using fc_model_weight_path :saved_models/fc_layers_caffe_vgg19_weights.hdf5
DEBUG:__main__:using test_result :saved_models/fc_layers_caffe_vgg19_result.npz
2018-03-13 21:52:51.780388: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1070, pci bus id: 0000:05:00.0, compute capability: 6.1)
2018-03-13 21:52:54.451871: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 2.00G (2147483648 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2018-03-13 21:52:55.013604: W tensorflow/core/common_runtime/bfc_allocator.cc:217] Allocator (GPU_0_bfc) ran out of memory trying to allocate 3.45GiB. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory is available.
DEBUG:__main__:Prediction time :4355.044603212038
DEBUG:__main__:(368584, 3)
DEBUG:__main__:(368584, 3)
DEBUG:__main__:(368584, 3)
DEBUG:__main__:(368584, 3)
DEBUG:__main__:Test accuracy: 50.4194%
